lib/validators/liveValidator.ts

```diff
@@ -781,6 +781,26 @@ export class LiveValidator {
     return op as OperationWithApiVersion
   }
 
+  private async getMatchedPaths(jsonsPattern: string): Promise<string[]> {
+    const matchedPaths = await globby(jsonsPattern, {
```

Comment:
is it possible to directly pass an array of strings to globby? if so maybe we can avoid concating the swagger paths

Reply:
Good suggestion, changed it.